presto-native-execution/presto_cpp/main/types/VeloxToPrestoExpr.cpp

```diff
@@ -0,0 +1,331 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/types/VeloxToPrestoExpr.h"
+#include <boost/algorithm/string.hpp>
+#include "presto_cpp/main/types/PrestoToVeloxExpr.h"
+#include "velox/core/ITypedExpr.h"
+#include "velox/expression/ExprConstants.h"
+#include "velox/vector/ConstantVector.h"
+
+using namespace facebook::presto;
+
+namespace facebook::presto::expression {
+
+using VariableReferenceExpressionPtr =
+    std::shared_ptr<protocol::VariableReferenceExpression>;
+
+namespace {
+const std::string kVariable = "variable";
```

Comment:
Are these names from the presto_protocol ? Can we avoid defining these and using the variables directly from the protocol ?

Reply:
@aditi-pandit, these variables are not obtained from presto_protocol and protocol would need to be enhanced to obtain such string constants from Presto java. Can it be taken up in a follow-up PR, since this refactor would also impact existing constants throughout `presto-native-execution`?